### ISSUE TITLE
How to use Subscription Using AndroidBillingLibrary

### ISSUE THREAD

**Issue by RavinderPoonia** [robotmedia/AndroidBillingLibrary#89]
Hi ,

   I want to use subscription functionality in one of my application. I am using AndroidBiillingLibrary. Can   you please help me that how can i do using this Library. Thanx in advance .

**Comment by ludocrazy** [comment]
i'm new to using AndroidBillingLibrary and i may be mistaken, but as far as i understood the original author(s) the intention of this library is to open source "bugfixing" and help you having to write less code yourself.

you still have to carefully read and understand the official google documentation, which should then help you to quickly look through the source of this library.

http://developer.android.com/guide/google/play/billing/index.html

basically as you have already used requestPurchase() now look for requestSubscription().

if i am right, and you simply missed a detail from the docs, then please close this issue! thanks :-)
